rr_description/urdf/sedani.urdf

```diff
@@ -321,7 +321,7 @@
     <joint name="base_footprint_to_chassis" type="fixed">
         <parent link="base_footprint"/>
         <child link="chassis"/>
-        <origin xyz="0 0 0.036" rpy="0 0 0"/>
+        <origin xyz="0 0 0.079" rpy="0 0 0"/>
```

Comment:
Why is this value changing? Did we remeasure? I know with the camera joint we found certain values that made the transformation work better but this seems like it will affect more than the camera.

Reply:
Yeah sedani.urdf used to be a verbatim copy of macaroni.urdf and so far things have been changing only as needed. This is the real height of Sedani's back axle. I think the lidar joint has been updated, but the IMU joint has not